ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/platform_settings.schema.yml

```
@@ -90,6 +90,8 @@ keys:
         raw_eos_cli:
           type: str
           description: EOS CLI rendered directly on the root level of the final EOS configuration.
+        security_entropy_source:
+          type: str
```

Can add description.
```suggestion
          type: str
          description: Entropy source improves the randomness of the numbers used to generate MACsec's cryptographic keys.
```